scripts/newrelic-cli.rb.tmpl

```diff
@@ -0,0 +1,34 @@
+class NewrelicCli < Formula
```

Comment:
A better place for this file would be `build/package/homebrew/`

Reply:
Won't be using this anymore, but good to know where to put those sorts of things 😎